### PR TITLE
feat: Add Mailing list analysis initial content

### PR DESCRIPTION
**Pull request by tumido**
Content will be available at https://www.operate-first.cloud/data-science/mailing-list-analysis/

### CODE REVIEW

**Review by MichaelClifford — APPROVED** [review]
LGTM

File: content/data-science/index.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ _This document contains a list of projects within the AI Ops Team at Red Hat._
 
     Github Repo : https://github.com/aicoe-aiops/sentiment-analysis-public
 
-8.  **Sync Pipelines** : Data ingress pipelines for DataHub via Argo pipelines.
+8. **Sync Pipelines** : Data ingress pipelines for DataHub via Argo pipelines.
 
     Github Repo : https://github.com/aicoe-aiops/sync-pipelines
+
+9. **Mailing list Analysis**: This analysis contains example code for how to develop a custom end-to-end email analytics service using the Open Data Hub on OpenShift. We demonstrate this on a text analysis on a Fedora mailing list. This list contains many discussions about the issues occurring with Fedora development on a monthly basis and suggestions for how to address the issues. This project aims to help the Fedora community bring a more data driven approach to their planning process by performing text analysis and gathering insights into the trends in the email conversations.
```
Comment (oindrillac) [inline content/data-science/index.md]: @tumido this list is alphabetically ordered. So could we move this item to number 5?

Also could we rephrase the second sentence as -
`We demonstrate this by performing text analysis on the Fedora mailing list.`

**Review by oindrillac — CHANGES_REQUESTED** [review]
Changes look good :tada: :+1: 
Added one comment.

File: content/data-science/index.md (hunk 1 above)
Comment (tumido) [reply content/data-science/index.md]: Done :+1:

**Review by oindrillac — APPROVED** [review]
:ship:

**Pull request merged**